example/provider-local/garden/base/cloudprofile.yaml

```diff
@@ -27,10 +27,6 @@ spec:
     - version: 1.0.0
       cri:
       - name: containerd
-      # provider-local image doesn't contain a full docker runtime but includes nerdctl imitating the docker CLI for fulfilling gardener's bootstrap needs:
```

Comment:
Hmm..Is it necessary? One should not ideally refer to the local cloudprofile for example.

Reply:
My argument to keep it would be that it's a helpful comment for future developers so they do not wonder why we use nerdctl, and maybe a TODO to remove nerdctl once the node agent is GA since I guess we won't need it then.
But no strong opinion, so its your call.

Reply:
I have added a task to remove `nerdctl` [here](https://github.com/gardener/gardener/issues/8023). I guess it's fine to keep the changes @shafeeqes proposed (i.e., not keep the comment), given that we want to promote the feature gate in the next weeks anyways.